sockeye/rerank.py

```diff
@@ -39,21 +39,32 @@ class Reranker:
 
     :param metric: Sentence-level metric such as smoothed BLEU.
     :param return_score: If True, also return the sentence-level score.
+    :param isometric_alpha: Factor for reranking with isometric criteria.
     """
 
-    def __init__(self, metric: str,
-                 return_score: bool = False) -> None:
-        if metric == C.RERANK_BLEU:
+    def __init__(self, metric: str, isometric_alpha: float, return_score: bool = False) -> None:
```

Comment:
you probably want to make isometric_alpha optional and the last argument? In Line 124 you pass the alpha as the last argument.